packages/blade/src/components/Chip/_decisions/decisions.md

```diff
@@ -0,0 +1,127 @@
+
+<img width="70%" src="./chips-variants.png" alt="Different variants of the Chip component">
+
+- [Design](#design)
+- [`Chip` API](#chip-api)
+- [`ChipGroup` API](#chipgroup-api)
+- [Examples](#examples)
+  - [Basic/Uncontrolled Usage](#basicuncontrolled-usage)
+  - [Controlled Usage](#controlled-usage)
+- [Accessibility Considerations](#accessibility-considerations)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=52377%3A27686&mode=design&t=pFzCay0x4a6sTcoQ-1) to all variants of the `Chip` component.
+
+## `Chip` API
+
+> **Note**
+>
+> The `Chip` component is inherently tied to the `ChipGroup` and cannot be utilized outside its context.
+
+| Prop                    | Type                                                                 | Default     | Description                                                                                                                                        | Required |
+| ----------------------- | -------------------------------------------------------------------- | ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| **children**            | `string`                                                             | `undefined` | Renders the label text for the Chip                                                                                                                | ✅       |
+| **icon**                | `Icon`                                                               | `undefined` | Displays the Blade Icon component within the Chip                                                                                                  | ❌       |
+| **isDisabled**          | `boolean`                                                            | `false`     | Disables or enables the Chip                                                                                                                       | ❌       |
+| **size**                | `'xsmall' \| 'small' \| 'medium' \| 'large'`                         | `'small'`   | Specifies the size of the rendered Chip                                                                                                            | ❌       |
+| **value (html native)** | `string`                                                             | `undefined` | The value of the input field in Chip, [useful in form submissions](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#value) | ❌       |
+| **variant**             | `'positive' \| 'negative' \| 'notice' \| 'information' \| 'neutral'` | `undefined` | Sets the Chip's visual variant                                                                                                                     | ❌       |
```

Comment:
Should this be `intent` or `variant` 🤔

Reply:
`variant`, similar to `badge`.

Reply:
For Badge there was chance of `blue` or `indigo` coming up. We currently use `intent` on other components 🤔 

Here's details of that discussion - https://github.com/razorpay/blade/blob/master/packages/blade/src/components/Badge/_decisions/decisions.md#variant-prop

Reply:
> but badge had use case for intent+custom colors hence we made it variant here i don't see that

I would lean towards variant, intent is not scalable. 

Abhi nehi hain but bad mein we might have usecase for blue or white chips (similar to how we need white button/links). 
If we go with intent we will later have to make a breaking change, similarly how we had to do in Counter.

Reply:
True but problem is variant isn't scalable either because variant has a contexual meaning based on components.

E.g. if we have negative button, we might have both. 

```jsx
<Button variant="secondary" intent="negative" />
```

So not sure if we can use variant as default either 🤔